src/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang='en'>
+    <head>
+        <meta charset='utf-8'>
+        <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>
+        <title>de·crypto</title>
+    </head>
+    <body>
+        <header id="header" class='text-center'>
+            <h1>de·crypto</h1>
+            <nav id="header-nav">
+                <ul class='nav justify-content-center'>
+                    <li class='navitem'>
+                        <a class='nav-link' href=''>projects</a>
+                    </li>
+                    <li class='navitem'>
+                        <a class='nav-link' href=''>team</a>
+                    </li>
+                    <li class='navitem'>
+                        <a class='nav-link' href=''>partners</a>
+                    </li>
+                </ul>
+            </nav>
+            <p>decentralized academic research group
+               for the political applications of cryptography</p>
+            <a href='' class='btn btn-primary'>Meet the team</a>
+        </header>
+        <main class='container'>
```

Comment:
Leave some space between header and main (`<br> <br/>` maybe?)

Reply:
Of course. I think is better to leave aesthetic changes for another PR. You have to decide colors, fonts etc before we can proceed. What do you think?

Reply:
LGTM